src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/OrderingPartyController.cs

```diff
@@ -44,7 +44,7 @@ public async Task<ActionResult<OrderingPartyModel>> GetAsync(string orderId)
 
             var result = new OrderingPartyModel
             {
-                Organisation = new OrganisationModel
+                Organisation = primaryContact is null ? null : new OrganisationModel

```

Comment:
Do we not want to return anything if we have no primary contact details or is having no primary contact details not considered a valid scenario?

Reply:
I think the only mandatory data we can be sure of is the primary contact information as this is entered through the order form application. The Organisation information is retrieved via the ODS service. Can we be sure the address information will be set? The basic requirement at the moment is if the Front end would like an empty object if no ordering party information is set. This is used to determine whether or not to make a request to the Organisation API.

Reply:
As discussed on the call, I think it might be worth refactoring this check to a method with a name like `HasMandatoryData` or `IsSectionComplete` to hopefully provide some additional context as to why we return nothing if we have no primary contact details.

Reply:
👍  Like the giving additional context approach by including a method name.

Reply:
👍